envs/staging-preview/.envrc

```diff
@@ -0,0 +1,44 @@
+export CHAIN_ID=1
+export THRESHOLD_NUMERATOR=2
+export THRESHOLD_DENOMINATOR=3
+export GENESIS_COMMITTEE_UTXO="4e65eddc6132c1b9891d43f9d67a4a0a7b6e614b648f59dc97538f3f23eb96bf#1"
+# cardano-cli address key-hash --payment-verification-key-file <path to authority vkey>
+export GOVERNANCE_AUTHORITY=1f0977bc0f57c67ca6d77296c1b575fe05a6dc2c5fa38056ba63c50c
+
+# unique identifier of the beneficiary that will be credited
+# producer rewards for blocks produced by this node
+export SIDECHAIN_BLOCK_BENEFICIARY="0x0000000000000000000000000000000000000000000000000000000000000201"
+
+# below exports parse config/addresses.json file, which needs to regenerated for each new sidechain by running
+# sidechain-main-cli addresses ... > config/addresses.json
+export COMMITTEE_CANDIDATE_ADDRESS=$(jq -r '.addresses.CommitteeCandidateValidator' envs/staging-preview/addresses.json)
+export D_PARAMETER_POLICY_ID=$(jq -r '.mintingPolicies.DParameterPolicy' envs/staging-preview/addresses.json)
+export PERMISSIONED_CANDIDATES_POLICY_ID=$(jq -r '.mintingPolicies.PermissionedCandidatesPolicy' envs/staging-preview/addresses.json)
+
+# native token observability
+export NATIVE_TOKEN_POLICY_ID='ada83ddd029614381f00e28de0922ab0dec6983ea9dd29ae20eef9b4'
+export NATIVE_TOKEN_ASSET_NAME='5043546f6b656e44656d6f'
+export ILLIQUID_SUPPLY_VALIDATOR_ADDRESS='addr_test1wrhvtvx3f0g9wv9rx8kfqc60jva3e07nqujk2cspekv4mqs9rjdvz'
+
+# Preview parameters
+export CARDANO_SECURITY_PARAMETER=432
+export CARDANO_ACTIVE_SLOTS_COEFF=0.05
+export DB_SYNC_POSTGRES_CONNECTION_STRING="postgres://postgres:password123@localhost/cexplorer"
+
+# A minimum block distance from the most recent MC stable block.
+# Used by block producers to select the stable block to include in the block header.
+export BLOCK_STABILITY_MARGIN=0
+
+# Timestamp for the MC_FIRST_EPOCH_NUMBER
+# Genesis should not have a timestamp before this one, this should be divisible by both sidechain slot and epoch durations
+export MC__FIRST_EPOCH_TIMESTAMP_MILLIS=1666656000000
+# First Shelley epoch number on Cardano
+export MC__FIRST_EPOCH_NUMBER=0
```

Comment:
Are you sure this is valid? I think that on preview there are non Shelley epochs. Please verify it before merging!

Reply:
Hmm I don't know, this was already configured in the former staging environment

Reply:
I've combed through preview genesis files and node configs, and as I understand the node is set to start in Shelley era immediately at epoch 0:

`  "TestShelleyHardForkAtEpoch": 0,`

There is nothing in the genesis files to indicate otherwise

I think these staging preview values are correct, as we have been using them already for a long time

The staging preprod .envrc values in this branch will be incorrect, but that is fine. I will loop back to that, I just need CI generating chain-specs for now

Reply:
My comment is in bad place. Preview is okay. I should have commented on preprod file. I missed in the diff, that this new file is preview but the moved file is preprod.

Reply:
Oh okay, yes I will come back to this in a second PR if that is okay. For now just getting the CI scaffolding of the preprod environment up and running, and then will work on final .envrc and keystores

Reply:
Of course it is. I'm doing this review for you and SETs. I won't be using it by myself.